### PR TITLE
Instructor: FeedbackEditPage: fix update message #7252

### LINKED ISSUE #7252: Instructor: FeedbackEditPage: fix update message
On instructorFeedbackEditPage, after editing a feedback question, the following message is shown to the user:

The changes to the question has been updated.

This should be changed to "The changes to the question have been updated.". Alternatively, it might be possible to rephrase this message to be clearer instead.

### PR DESCRIPTION
**Pull request by adityavhegde**
Fixes #7252 

The text that appears, after a question has been edited -- in the Instructor Feedback Session Edit page -- has been changed to "The changes to the question have been updated."
FEEDBACK_QUESTION_EDITED field in Const.java was changed to the above required text.
This successfully, changes the text value as expected.

Note: The edit button seems not to be working, after the I included most recent commits to my branch.

### CODE REVIEW

**Comment by chowyb** [comment]
Hi @adityavhegde,

Did you run `npm run build` after merging into your branch?

**Comment by adityavhegde** [comment]
Hi @chowyb , 
I am not sure I get you. Do you mean, after merging remote changes in my branch (i.e after syncing my repo with the teammates one)?

**Comment by damithc** [comment]
> I am not sure I get you. Do you mean, after merging remote changes in my branch (i.e after syncing my repo with the teammates one)?

Yes @adityavhegde . You need to run that command whenever there is a possible change to front-end code.

**Comment by adityavhegde** [comment]
Thank you for the feedback @damithc . 
I ran the `npm run build` (this fixes my issue of not being able to edit the questions) 
What are my next steps? 
I ran the CI test locally and there still seem to be failing test cases.

**Comment by damithc** [comment]
How many cases are failing? If it is fewer than 5, you can proceed to the next step in our Contributor Orientation Guide.

**Comment by damithc** [comment]
> How many cases are failing? If it is fewer than 5, you can proceed to the next step in our Contributor Orientation Guide.

@adityavhegde, this answer is not correct. I didn't you asked that question in the context of a PR. Let's wait to see if another dev can give you some helpful inputs on this PR.

**Comment by shubham49** [comment]
@adityavhegde Now you need to run test cases with god mode turned on. If you don't know about god mode, then take a good look at `godmode.md`. You can turn god mode on in `test.properties.` 
Just run cases with god turned on, commit the changes and push it.

**Comment by adityavhegde** [comment]
Thank you all for the feedback. With the God mode turned on, almost all cases have passed. I can still see a few cases (~5) that have failed (CI Test: ./gradlew ciTests).
Also my branch now shows many of these additional files that have been modified. 
Have these files in test been modified due to the God mode.
image

I ran the diff command to check the difference and this is what mostly is seen:
image

**Apologies**, if some of this is quite obvious. This is my first PR here. 
Should I add and commit these changes to the branch?

**Comment by VamsiSangam** [comment]
Yes, God mode is used to update the test files. But when you check the differences, you should be able to explain the changes. For example, in your second image, it shows some time getting modified. That change in time is probably not what you expect because you didn't change the code in that way. So, when you run god mode, make sure the changes happening in the test files are what you expect them to be.

Btw, the test environment should be using UTC time zone, please check if you have changed it [according to the docs](https://github.com/TEAMMATES/teammates/blob/master/docs/development.md#testing). That unexpected change test files about dates usually occurs because of this.

**Comment by adityavhegde** [comment]
Hi @VamsiSangam , thank you for the feedback. Yes as suspected, too many files modified were due to time zone not set to UTC while running CI Test. Upon re-running with correct time zone, I see the file changes as foll: 
image

As mentioned in the previous discussions, my guess is that, these changes were as a result of running the tests in God mode. They are inline, with what I expect them to be -- change in the text field as required.

I checked the travis-ci failed test cases. My change (in Const.java) was "The changes to the question have been updated." While the previous text was: "The changes to the question has been updated." The test files, still have the previous texts. Running in the God mode, has updated them.
Which means, I need to push these modified files, for the tests to pass. Should I go ahead and commit these files to the main branch ?

**Comment by VamsiSangam** [comment]
Yes you need to commit and push the test files also to the PR :blush:

**Comment by adityavhegde** [comment]
@VamsiSangam one question. Since my branch is a few commits behind the master, a git merge on the branch is required, yes ?

**Comment by VamsiSangam** [comment]
Well it is kind of optional actually :sweat_smile: . It is recommended to keep your PR as updated as possible, but it is really optional. I don't think most devs update their PR as soon as their master branch gets behind. They do it once every 2-3 days or so. And once your PR gets accepted we will sync this PR before it is merged. You don't have to do it.

But if there has been a recent addition, like say some style checks have been added, in that case you must update your fork and run those checks locally, else all tests would run locally but the Travis CI build might fail. :stuck_out_tongue_closed_eyes: 

Long story short, you don't have to do it. If it is critical, we will ask you. :smile:

**Comment by adityavhegde** [comment]
@VamsiSangam Thank you :)

**Comment by VamsiSangam** [comment]
Your Travis CI build is failing because you haven't yet pushed those modified test cases yet. Eg. `instructorFeedbackConstSumRecipientQuestionEditSuccess.html`. Please commit and push the test files modified by god mode.

Generally, in most cases Travis build doesn't fail just because your PR is not up to date with `TEAMMATES:master`

**Comment by adityavhegde** [comment]
@VamsiSangam Yes, I haven't pushed changes to the branch. In progress.
